lib/basedriver/image-helpers.js

```diff
@@ -0,0 +1,75 @@
+import { Buffer } from 'buffer';
+import { PNG } from 'pngjs';
+import B from 'bluebird';
+
+async function cropBase64Image (base64EncodedImage, rect) {
```

Comment:
It would be useful to add doctrings on these all helper methods Check https://github.com/appium/appium-adb/blob/9ee5b9251fb15cc2c0ae3e86b6d1659d6aa576cf/lib/tools/adb-commands.js on docstring examples